### DISCUSSION TITLE
Netdata updates

### DISCUSSION THREAD

**Discussion by JGamache-3144** [Homebrew/discussions#1212] [Casks]
How soon will the latest Netdata show up so I can update my Mac system?

Thanks

**Comment by carlocab** [comment]
As soon as someone opens a pull request to update the formula, and the pull request is merged after it passes CI.

Perhaps you'd like to open one. Here's our [contributing guide](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md#to-submit-a-version-upgrade-for-the-foo-formula). There's also a more detailed guide in the [docs](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request).
